lib/components/Editor/CustomExtensions/SlashCommands/ExtensionConfig.js

```diff
@@ -33,19 +34,19 @@ export default {
   }) => {
     let commandItems = [
       {
+        optionName: ADD_ON_OPTIONS.FONT_SIZE,
         title: "Paragraph",
         description: "Add a plain text block",
```

Comment:
Can we move the title and descriptions to constants as well? 
Also, please add periods to the end of each sentence in descriptions.